.gitignore

```diff
@@ -8,6 +8,9 @@ node_modules
 build
 dist
 
+# flow
+!dist/index.js.flow
```

Comment:
This doesn't seem quite right. I don't think anything in the "dist" folder should be committed to Git. Too likely to be stale.